src/v2/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertDeleteModal.tsx

```diff
@@ -0,0 +1,96 @@
+import { Text, Flex, ModalDialog, Button } from "@artsy/palette"
+import { useState } from "react"
+import createLogger from "v2/Utils/logger"
+import { useDeleteSavedSearchAlert } from "../useDeleteSavedSearchAlert"
+import { Media } from "v2/Utils/Responsive"
+
+const logger = createLogger(
+  "v2/Apps/SavedSearchAlerts/Components/SavedSearchAlertDeleteModal"
+)
+
+interface SavedSearchAlertDeleteModalProps {
+  id: string
+  onCloseClick: () => void
+  onDeleted: () => void
+}
+
+export const SavedSearchAlertDeleteModal: React.FC<SavedSearchAlertDeleteModalProps> = ({
+  id,
+  onCloseClick,
+  onDeleted,
+}) => {
+  const [isDeleting, setIsDeleting] = useState(false)
+  const { submitMutation: submitDeleteAlert } = useDeleteSavedSearchAlert()
+
+  const deleteAlertById = async () => {
+    try {
+      setIsDeleting(true)
+      await submitDeleteAlert({
+        input: {
+          searchCriteriaID: id,
+        },
+      })
+    } catch (error) {
+      logger.error(error)
```

Comment:
**question(asking cause I don't have context on this):** is this the default way to handle errors in force? Do we collect these in sentry?

Reply:
To be honest, I'm not really in the context either, but as far as I see [here](https://github.com/artsy/force/blob/8d6e5277b254d4aa70cb7428c5986b4e56b21e82/src/v2/Utils/logger.ts#L21) - yes, the error will be sent to Sentry